src/state/editor.js

```diff
@@ -1,7 +1,7 @@
 import { DOMSerializer } from "prosemirror-model";
 import { Container } from "unstated";
 import { prettyPrint } from "html";
-import nanoid from "nanoid";
+import { nanoid } from "nanoid/non-secure";
```

Comment:
i was getting a browser error for using 'secure' nanoid in a browser, and this is the solution. it required updating nanoid.